src/app/lib/config/services/news.js

```diff
@@ -4,6 +4,7 @@ import { latin } from '@bbc/gel-foundations/scripts';
 import * as fonts from '@bbc/psammead-styles/fonts';
 
 const news = {
+  lang: 'en_GB',
```

Comment:
```suggestion
  lang: 'en-GB',
```
The lang value should have a dash rather than an underscore, as in the previous comments. Everything else here looks good.